src/SmartComponents/ActivityTable/__tests__/ActivityTable.tests.js

```diff
@@ -102,9 +118,61 @@ describe('CompletedTasksTable', () => {
     expect(input.value).toBe('');
   });
 
+  it('should filter by status completed', async () => {
+    fetchExecutedTasks.mockImplementation(async () => {
+      return activityTableItems;
+    });
+
+    render(
+      <MemoryRouter keyLength={0}>
+        <Provider store={store}>
+          <ActivityTable />
+        </Provider>
+      </MemoryRouter>
+    );
+
+    await waitFor(() => expect(fetchExecutedTasks).toHaveBeenCalled());
```

Comment:
I believe we can use a single waitFor for all the `expect`s. In other words, we can group all the expected asynchronous calls inside a single waitFor.

Reply:
This works for most cases, but there is a test in this file that fails, so I left the test as-is.